### DISCUSSION TITLE
Shadows on Bootstrap page

### DISCUSSION THREAD

**Discussion by AFireInside777** [twbs/bootstrap#39752] [Q&A]
Hello. Not sure if this is where this kind of thing is posted, but on your page for Shadows: https://getbootstrap.com/docs/4.1/utilities/shadows/, none of the examples provided show any shadows.
I'm not sure if this is something wrong with my browser or the page.

**Comment by julien-deramond** [comment]
Hello @AFireInside777 
No problem with the shadows on my side with different combinations of browsers and operating systems, I can see them perfectly:

<img width="887" alt="Screenshot 2024-03-06 at 21 37 11" src="https://github.com/twbs/bootstrap/assets/17381666/0cc771dd-078e-4039-9c39-4e3185d720f5">

By inspecting the code, you'll also see that the utilities (classes) also define the shadows.

It could be, IDK, the lightness settings of your monitor maybe; the light gray might be difficult to perceive.
